### PR TITLE
Fix the example

### PR DESCRIPTION
**Pull request by gaearon**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,11 +2,13 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="viewport" content="width=400, height=600, initial-scale=0.8, maximum-scale=1">
```
Comment (gaearon) [inline index.html:5]: Got this from Codepen settings box

File: index.html [hunk 2]
```diff
@@ -2,11 +2,13 @@
...
     <title>React App</title>
   </head>
   <body>
-    <div id="root"></div>
+    <div id="wrapper">
```
Comment (gaearon) [inline index.html:9]: This is what Michael used in Codepen and styles reference these IDs

File: src/Calculator.js [hunk 3]
```diff
@@ -1,8 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import ReactPoint from 'react-point';
-
-const PointTarget = ReactPoint.PointTarget
+import { PointTarget } from 'react-point';
```
Comment (gaearon) [inline src/Calculator.js:3]: This is from `react-point` README: it wants to be a named import

File: src/Calculator.js [hunk 4]
```diff
@@ -1,8 +1,7 @@
...
+import './Calculator.css';
```
Comment (gaearon) [inline src/Calculator.js:4]: Don’t forget the CSS!

**Comment by mjackson** [comment]
💖 Thanks, @gaearon! :D

**Comment by aaronplanell** [comment]
Thanks @gaearon !

**Pull request merged**